examples/api/test/material/chip/deletable_chip_attributes.on_deleted.0_test.dart

```diff
@@ -0,0 +1,65 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/chip/deletable_chip_attributes.on_deleted.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  Finder cancelIconFinder(String chipText) {
+    return find.descendant(
+      of: find.widgetWithText(Chip, chipText),
+      matching: find.byIcon(Icons.cancel),
+    );
+  }
+
+  testWidgets('Chip.onDeleted can be used to delete chips', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const example.OnDeletedExampleApp(),
+    );
```

Comment:
```suggestion
  testWidgets('Chip.onDeleted can be used to delete chips', (WidgetTester tester) async {
    await tester.pumpWidget(
      const example.OnDeletedExampleApp(),
    );
    
  Finder cancelIconFinder(String chipText) {
    return find.descendant(
      of: find.widgetWithText(Chip, chipText),
      matching: find.byIcon(Icons.cancel),
    );
  }
    
```


Unless the code is reused in multiple tests, it's better to keep inside the test code for readability and portability.